src/bld.js

```diff
@@ -270,7 +270,7 @@ const manageManifest = async ({ nwPkg, managedManifest, outDir, platform }) => {
   );
 
   process.chdir(
-    path.resolve(
```

Comment:
This looks like premature optimization. Unless you can measure the actual benefits of this and they are substantial, I don't think it justifies reducing code readability. For example `resolve` is a word commonly associated to promises. `(resolve, reject)`. So seeing it by itself I would assume that is what it relates to. However, seeing it attached to the `path` import, I know that it actually gives me a resolved file path. This goes with all the rest, seeing the associated hierarchy of the import is generally helpful while working on a code base.

Readability and maintainability are more important than build time or memory usage.

What impact would the performance increase have? If it only shaves off a few seconds of a multi-minute process, it's not worth doing.

I wouldn't mind faster builds, but I think we would need to focus on actual architectural and 3rd-party dependency evaluation over saving a few KB of memory by skipping importing a native node module.

Reply:
This certainly saves a few kb on launch but during use it has an effect allowing you to save a few seconds. I can understand that this is a small impact after all depends on the number of builds you launch in a row 😄 

For the Resolve part I was careful not to end up in promises that could have posed a potential conflict